lib/scout_apm/config.ex

```diff
@@ -10,15 +10,20 @@ defmodule ScoutApm.Config do
 
   use GenServer
 
+  require Logger
+
+  alias ScoutApm.Config.Coercions
+
+  @name __MODULE__
+
   ## Client API
 
   def start_link do
-    GenServer.start_link(__MODULE__, :ok, [name: __MODULE__])
+    GenServer.start_link(@name, :ok, [name: @name])
```

Comment:
I learned this trick since writing this code the first time. Name, and then referring to it by name rather than doing `Process.whereis` silliness.